snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml

```diff
@@ -165,3 +165,372 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.3375.2.1.1.2.9.29
       name: sysClientsslStatHandshakeFailures
+
+  # Metrics from F5-BIGIP-LOCAL-MIB provide monitoring information about virtual servers, nodes, and pools.
+  # These concepts are related to the LTM (Local Traffic Management) features of BIG IP devices.
+  # TL;DR:
+  # * Virtual servers act as load balancers that are assigned pools of nodes (= servers).
+  # * Clients can use a virtual server as an entrypoint to an application service that spans multiple machines.
+  # Learn more: https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-basics-11-6-0.html
+
+  # Virtual servers.
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    symbol:
+      OID: 1.3.6.1.4.1.3375.2.2.10.1.1
+      name: ltmVirtualServNumber
+
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.1.2
+      name: ltmVirtualServTable
+    symbols:
+      - OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.9
+        name: ltmVirtualServEnabled
+      - OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.10
+        name: ltmVirtualServConnLimit
+    metric_tags:
+      - tag: server
```

Comment:
Actually, I believe I initially had this one in, same for `ltmNodeAddrAddr`.

But then figured it might be a bit problematic to submit tags referring to IP addresses, mainly for doubts regarding high cardinality.

In theory we'd have 1 IP per server, but I don't know if there are cases where users would rotate IPs for whatever reason while keeping the underlying server the same.

I may be mistaken/overthinking this, and getting metrics from a specific IP address is probably useful to users. WDYT?

FWIW, I looked at other profiles and the only example of submitting an `ip_address` tag is in Dell Poweredge:

https://github.com/DataDog/integrations-core/blob/d6add1dfcd99c3610f45390b8d4cd97390af1f69/snmp/datadog_checks/snmp/data/profiles/dell-poweredge.yaml#L151-L154